Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.xaml

```diff
@@ -0,0 +1,734 @@
+        <Setter Property="ScrollViewer.IsVerticalRailEnabled" Value="False" />
+        <Setter Property="ScrollViewer.ZoomMode" Value="Disabled" />
+        <Setter Property="ScrollViewer.IsDeferredScrollingEnabled" Value="False" />
+        <Setter Property="ScrollViewer.BringIntoViewOnFocusChange" Value="True" />
+        <!--  Need UseLayoutRounding as Width being dynamically changed in same cases to fill up an entire space, otherwise gap.  -->
+        <Setter Property="UseLayoutRounding" Value="False" />
+        <Setter Property="ItemContainerTransitions">
+            <Setter.Value>
+                <TransitionCollection>
+                    <AddDeleteThemeTransition />
+                    <ContentThemeTransition />
+                    <ReorderThemeTransition />
+                    <EntranceThemeTransition IsStaggeringEnabled="False" />
+                </TransitionCollection>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="ItemsPanel">
+            <Setter.Value>
+                <ItemsPanelTemplate>
+                    <!--  Note: We have to use StackPanel here due to other issues using ItemsStackPanel or VirtualizingStackPanel  -->
+                    <StackPanel Orientation="Horizontal" />
+                </ItemsPanelTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Template" Value="{StaticResource TabViewTemplate}" />
+    </Style>
+
+    <ControlTemplate x:Key="TabViewTemplate"
+                     TargetType="local:TabView">
+        <Grid x:Name="TabViewContainer">
+            <Grid.RowDefinitions>
+                <!--  0 Header  -->
+                <RowDefinition Height="Auto" />
+                <!--  1 Tabs  -->
+                <RowDefinition Height="Auto" />
+                <!--  2 Content  -->
+                <RowDefinition Height="*" />
+                <!--  3 Footer  -->
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <!--  0 TabStartHeader  -->
+                <ColumnDefinition Width="Auto" />
+                <!--  1 Tabs  -->
+                <ColumnDefinition Width="Auto"
+                                  local:TabView.ConstrainColumn="True"
+                                  local:TabView.IgnoreColumn="True" />
+                <!--  2 Reserved: Overflow Drop-down  -->
+                <ColumnDefinition Width="Auto" />
+                <!--  3 Auto Add Button  -->
+                <ColumnDefinition Width="Auto" />
+                <!--  4 Padding  -->
+                <ColumnDefinition Width="*"
+                                  local:TabView.IgnoreColumn="True" />
+                <!--  5 TabEndHeader  -->
+                <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+            <ContentPresenter Grid.Row="0"
+                              Grid.ColumnSpan="7"
```

Comment:
We only have five columns

Reply:
There's 6 columns, I was just fixing this one.